public/app/percona/settings/Settings.service.ts

```diff
@@ -21,14 +21,14 @@ export const SettingsService = {
       const { settings } = await api.put<SettingsAPIResponse, Partial<SettingsAPIChangePayload>>(
         '/v1/server/settings',
         body,
-        false,
+        true,
```

Comment:
Just, noticed, could you pass the disableNotifications through a parameter? (like with `getSettings`). Then you could pass true just where the setSettings call is made (if we for whatever reason need the original notifications available)